docs/guide/controller-programming.md

```diff
@@ -452,6 +452,19 @@ if (finished) {
 
 Note that the exit status as well as the value returned by the `main` function are ignored by Webots.
 
+### Console Output
+
+As mentionned above, outputting something from the controller will be redirected by Webots to the console. The Webots console can not be used interactively but still supports some basic ANSI sequences. One can output text with certain style, using color for example.
+
+Currently the console supports the following:
+  - 8 bit color (foreground and background)
+  - Bold style
+  - Underline style
+  - Clear screen (same as issuing `clear` command in your terminal)
+  - Resetting style
+
+There is a world example in webots: samples/devices/console_capabilities or at `$WEBOTS_HOME/projects/samples/devices/controllers/console_capabilities` to demonstrate how to use those.
```

Comment:
I would rename this example to `console` which is less verbose than `console_capabilities` and is more consistent with other device examples.